src/components/footer/Footer.module.scss

```diff
@@ -1,30 +1,75 @@
 @import '../../utils/variables';
 
 .footer {
-  padding: 40px;
+  padding: 40px 100px 100px 100px;
   display: flex;
-  flex-wrap: wrap-reverse;
-  justify-content: space-between;
+  // flex-wrap: wrap-reverse;
+  justify-content: center;
   font-size: 28px;
+  background-color: #FFF8F1;
   & a {
     color: $black;
     padding: 0 20px;
   }
 }
 
-.links {
+.footerRight {
   font-size: 18px;
+  width: 50%;
+  text-align: center;
+}
+
+.footerLeft {
+  display: flex;
+  justify-content: flex-start;
+  width: 50%;
+  display: flex;
 }
 
+.logo {
+  width: 150px;
+  padding-left: 120px;
+  padding-top: 60px;
+}
+.linksColumnOne {
+  width: 100%;
+}
+.linksColumnTwo {
+  width: 100%;
+}
 .socialLinks {
   display: flex;
+  justify-content: center;
 }
 
-@media (max-width: $small) {
-  .links {
-    margin-top: 24px;
-  }
+.footerBackground {
+  background-color: #FFF8F1;
+}
+
+.links {
+  font-size: 20px;
+  padding-bottom: 20px; 
+}
+
+
+.appDownloadLinks {
+  margin: 0 -16px 20px 0 ;
+}
+@media (max-width: 900px) {
   .footer {
-    justify-content: center;
+    display: list-item;
+    width: auto;
+    padding: 50px 0 50px 0
+  }
+  .footerLeft {
+    width: 100%;
+    text-align: center;
+  }
+  .footerRight {
+    width: 100%;
+  }
+  .logo {
+    margin: auto;
+    padding: 50px 0 0 0  
   }
 }
```

Comment:
This is again similar to the comment on #58 but I think nesting really helps keep the code clean (feel free to say Im wrong). But in this case I would have media queries done in the same way as @tunangst has done in his update so: 
```
.profile-pic {
  float: left;
  width: 250px;
  @media screen and (max-width: $break-small) {
    width: 100px;
    float: none;
  }
```
which compiles to:
```
.profile-pic {
  float: left;
  width: 250px;
}
@media screen and (max-width: 320px) {
  .profile-pic {
    width: 100px;
    float: none;
  }
}
```
http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32

Similarly, I think things like .footerRight and .footLeft if those are always children of .footer then tose can be nested within the .footer block

Reply:
I am happy to nest the @media inside of their respective elements - though I find when writing the CSS for dynamic page sizes, its easier to have all the css code together for each screen size (if there are multiple) as it makes it easier to adjust?

Reply:
So, I am not sure if this has been discovered but if you nest the css elements, the responsive media size cant read it (even if theyre nested in there, as well), so we will probably have to keep it separated. I hope that is OK!

Reply:
If you find a way around this PLEASE let me know as im interested in how this works!!!